### PR TITLE
added header back button to system icons

### PR DESCRIPTION
**Pull request by idango10**

### CODE REVIEW

File: packages/wix-ui-icons-common/src/system/raw/HeaderBackButton.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" viewBox="0 0 30 30">
+  <path fill="#17191C" d="M12.5,16.2151489 L11.7928932,15.5080421 L17.4497475,9.84314575 L18.1568543,10.5502525 L12.5,16.2151489 Z M11.7942505,15.5 L12.5013573,14.7928933 L18.1568543,20.449747 L17.4497475,21.1568538 L11.7942505,15.5 Z"/>
```
Comment (shlomitc) [inline packages/wix-ui-icons-common/src/system/raw/HeaderBackButton.svg:2]: please remove the `fill` property.

File: packages/wix-ui-icons-common/src/system/raw/HeaderBackButton.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" viewBox="0 0 30 30">
```
Comment (shlomitc) [inline packages/wix-ui-icons-common/src/system/raw/HeaderBackButton.svg:1]: move this file to the relevant folder `wix-ui-icons-common/src/editor-classic/system/raw`

File: packages/wix-ui-icons-common/src/system/metadata.ts [hunk 3]
```diff
@@ -147,6 +147,13 @@ const systemIconsMetadata: Array<IconMetadata> = [
     tags: [],
     sizes: { "18x16": "FormFieldWarningFilled" },
   },
+  {
```
Comment (shlomitc) [inline packages/wix-ui-icons-common/src/system/metadata.ts:150]: move this definition to the relevant folder `wix-ui-icons-common/src/editor-classic/system/raw`

**Review by shlomitc — CHANGES_REQUESTED** [review]
Thanks for this PR!
I wrote a few comments.
Please also take the time to [sign your commits](https://github.com/wix-private/wix-design-systems/blob/master/packages/wix-style-react/docs/contribution/CREATE_PR.md#sign-commits), CI won't run otherwise

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-icons-common**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-icons-common.pr_2113/

**Comment by shlomitc** [comment]
perfect, thanks!

**Pull request merged**